app/src/pages/inside/common/utils.js

```diff
@@ -169,6 +172,9 @@ export const getGroupedDefectTypesOptions = (
           color: defectType.color,
           locator: defectType.locator,
           meta: {
+            groupRefLabel: defectTypesLocalization[defectType.typeRef.toLowerCase()]
+              ? formatMessage(defectTypesLocalization[defectType.typeRef.toLowerCase()])
+              : defectType.longName,
             longName: defectType.longName,
             shortName: defectType.shortName,
```

Comment:
Seems like the `shortName` used together with `groupRefLabel` every time.
Can we join these two properties into one?

Reply:
It used together here, but FE in QA we don't need `groupRefLabel`